Sources/FluentDarkModeKit/DarkModeManager.swift

```diff
@@ -4,6 +4,9 @@
 //
 
 import UIKit
+#if SWIFT_PACKAGE
+import DarkModeCore
```

Comment:
Could we leverage `@_exported import` in here? 

In this way the user of this Swift PM can just `import FluentDarkModeKit` and don't need to `import DarkModeCore`. And we can delete other `import DarkModeCore` in extensions files.

Reply:
Thanks for pointing it out. I think it is good to use `@_exported` to unify the behavior between integrating via SPM and via other tools, the only part that concerns me is that this `@_exported` is non public (although I suppose some devs are already using it). what do you think? @icodesign

Reply:
I think it's OK to use.